doc/user_guide/lfric.rst

```diff
@@ -1961,15 +1961,27 @@ operates_on                    Data passed for each field/operator argument
 ``cell_column``                Single column of cells from the owned region (except
                                when performing an INC operation on continuous fields
 			       when it will include one level of halo cells).
+``owned_cell_column``          As ``cell_column`` but the iteration space must *only*
+                               cover owned columns (i.e. excluding halo columns).
 ``halo_cell_column``           Single column of cells exclusively from halo region.
 ``owned_and_halo_cell_column`` Single column of cells but iteration space will include
                                both owned and halo regions.
-``dof``                        Single DoF .
+``dof``                        Single DoF.
+``owned_dof``                  Single DoF but the iteration space must *only* include
+                               owned DoFs (i.e. not annexed or halo DoFs).
 ``domain``                     All columns of cells in the (sub-)domain.
 ============================== =======================================================
 
-(For a description of the concepts of 'owned' and 'halo' cells please see the
-:ref:`lfric-developers`.)
+(For a description of the concepts of 'owned' and 'halo' cells and 'annexed' DoFs
+please see the :ref:`lfric-developers`.)
+
+The ``owned_cell_column`` and ``owned_dof`` values of ``OPERATES_ON`` are intended for
+use only with special cases where the kernel concerned cannot be used to perform
+redundant computation (e.g. when filling a field with pseudo-random numbers without
+regard to cell location). Processing an application that makes use of such a kernel
+requires that the ``COMPUTE_ANNEXED_DOFS`` configuration option (see
+:ref:`lfric-annexed_dofs`) be set to ``False`` as PSyclone can no longer guarantee that
+annexed DoFs are always clean between different ``invoke`` calls.
```

Comment:
I don't understand this, shouldn't using the new "owned_dofs" operate in a similar way regarding as compute_anexed_dofs  as "owned_cell_columns" does to redundant computation? E.g. preventing computing annex dofs (requiring a halo exchange) for such kernels.